i18n/ru.toml

```diff
@@ -122,9 +122,6 @@ other = "Доля страниц, в %, загруженных по HTTPS (ск
 [certificate_transparency_name]
 other = "Название"
 
-[certificate_transparency_role]
-other = "Назначение"
-
```

Comment:
Why certificate_transparency_role_test/certificate_transparency_role_testing is not updated for `ru`?

Reply:
Good catch! I forgot to update that file.